core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -105,6 +109,7 @@ class KafkaApis(val requestChannel: RequestChannel,
                 val authorizer: Option[Authorizer],
                 val quotas: QuotaManagers,
                 val fetchManager: FetchManager,
+                val sharePartitionManagerOption: Option[SharePartitionManager],
```

Comment:
I don't see `Option` suffix for other optionals in this class:
```suggestion
                val sharePartitionManager: Option[SharePartitionManager],
```